pkg/expect/expect_test.go

```diff
@@ -40,3 +38,17 @@ func TestEcho(t *testing.T) {
 		t.Fatalf("expected error on closed expect process")
 	}
 }
+
+func TestSendLine(t *testing.T) {
+	ep, err := NewExpect("/usr/bin/tr", "a", "b")
```

Comment:
OK, I found "a b" works if /usr/bin/tr is symlinked to busybox, which is a surprise...